tests/Database/SelectFields/DepthTests/DepthTest.php

```diff
@@ -64,13 +64,9 @@ public function testDefaultDepthExceeded(): void
 
         $this->sqlCounterReset();
 
-        $result = $this->httpGraphql($graphql, [
-            'expectErrors' => true,
-        ]);
+        $result = $this->httpGraphql($graphql);
 
-        self::assertCount(1, $result['errors']);
-        self::assertSame('Internal server error', $result['errors'][0]['message']);
-        self::assertSame('SQLSTATE[HY000]: General error: 1 no such column: posts.user (SQL: select "posts"."id", "posts"."user", "posts"."user_id" from "posts" where "posts"."user_id" in (1) order by "posts"."id" asc)', $result['errors'][0]['debugMessage']);
+        self::assertCount(0, $result['errors'] ?? []);
```

Comment:
Testing for "absence of errors" is weird with the current way the `httpGraphql` works 😬: because _if_ there's an error, it will abort the execution already and never hit any assertions.

Nevertheless, I might think that it would make it more clear to use this, WDY?
```suggestion
        self::assertArrayNotHasKey('errors', $result);
```